src/Resources/translations/messages.en.yaml

```diff
@@ -7,6 +7,7 @@ setono_sylius_analytics:
         tracking_id: Tracking id
         tracking_id_placeholder: Insert your tracking id, i.e. UA-12345678-1...
         enabled: Enabled
+        disabled: Disabled
```

Comment:
Is this used anywhere?

Reply:
No necessary after parent Sylius template delegation. Removed in last commit.